titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/BasicContainer.java

```diff
@@ -77,20 +81,30 @@ public Image getImage() {
     }
 
     public List<String> getEntryPoint() {
-        if (entryPoint == null) { return Collections.emptyList();}
+        if (entryPoint == null) {
+            return Collections.emptyList();
+        }
         return entryPoint;
     }
 
     public List<String> getCommand() {
-        if (command == null) { return Collections.emptyList();}
+        if (command == null) {
+            return Collections.emptyList();
+        }
         return command;
     }
 
     public Map<String, String> getEnv() {
-        if (env == null) { return Collections.emptyMap();}
+        if (env == null) {
```

Comment:
Validation annotations should ensure that all values are non-null.

Reply:
Gotcha, I've removed these from command/env/entrypoint.